src/test/java/tests/FindValuesAndCompare.java

```diff
@@ -0,0 +1,34 @@
+package tests;
+
+import factoryPattern.PageTypeEnums;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static factoryPattern.PageTypeEnums.*;
+import static utils.extentreports.ExtentTestManager.startTest;
+
+@Epic("Search function validation")
+@Feature("Search function validation")
+public class FindValuesAndCompare extends BaseTest {
+
+    @DataProvider(name = "data-provider")
+    public Object[][] dpMethod() {
+        return new Object[][]{{GOOGLE, "Ukraine", "input", GOOGLE_STRATEGY, "window", "searchType"},
+                {BING, "EPAM", "input", BING_STRATEGY, "window", "searchType"}};
+    }
```

Comment:
not sure this is a benefit of using DataProvider due to a lot of parameters. It makes Tests unreadable. Let's discuss